client/src/App.js

```diff
@@ -1,10 +1,12 @@
 import Home from "./pages/Home";
 import { Routes, Route } from "react-router-dom";
+import CourseSearch from "./pages/student/CourseSearch";
 
 function App() {
   return (
     <Routes>
-      <Route path="/" element={<Home />} />
+      <Route exact path="/" element={<Home />} />
+      <Route path="/course-search/:name" element={<CourseSearch />} />
```

Comment:
Why not change the URL to simply search? Also, if possible, change the name as a query parameter.

Reply:
resolved issues